.gitignore

```diff
@@ -9,4 +9,7 @@
 
 # tool caches
 .mypy_cache/
-__pycache__/
\ No newline at end of file
+__pycache__/
+
+# Mac OS folder file
+.DS_Store
```

Comment:
Thanks!  This has been added now https://github.com/bennord/appstore-tools/pull/14